common/app/views/fragments/atoms/mediaAtomCaption.scala.html

```diff
@@ -4,7 +4,6 @@
 <figcaption class="@RenderClasses(Map(
     "caption" -> true,
     "caption--img" -> true,
-    "caption--media-atom" -> true,
```

Comment:
The rules associated with this class were removed in a previous PR so it can be safely deleted.